ereuse_devicehub/resources/device/models.py

```diff
@@ -106,6 +106,9 @@ class Device(Thing):
     image = db.Column(db.URL)
     image.comment = "An image of the device."
 
+    allocated = db.Column(Boolean, default=False)
+    allocated.comment = "An image of the device."
```

Comment:
Add the description of the allocated attribute into Device class within the comment field.